Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,20 @@
+const fs = require("fs");
+const input = fs.readFileSync("./input.txt", "utf8");
+function calculateFrequency(arg1){
+    let inputToArray = arg1.split('\n').map(Number);
```

Comment:
Can you explain how `.map(Number)` works? This is something relatively advanced but not difficult to understand.